### DISCUSSION TITLE
Search for libraries

### DISCUSSION THREAD

**Discussion by ProfLeao** [ReactionMechanismGenerator/RMG-Py#2211] [Q&A] [answered]
Dear, is there any tool to check the occurrence of a compound in RMG libraries (thermo, reaction, kinetics...)? I would like to search from smiles.

**Comment by mefuller** [comment, marked as answer]
This may be accomplished on the RMG website for individual species:
https://rmg.mit.edu/molecule_search
https://rmg.mit.edu/database/kinetics/search/

If you are looking to automate searching, perhaps with a long list of compounds, I know this should be possible, but have not personally implemented it
